src/librustc/mir/interpret/value.rs

```diff
@@ -452,6 +458,13 @@ impl<'tcx, Tag> Scalar<Tag> {
         Ok(b as i32)
     }
 
+    pub fn to_i16(self) -> InterpResult<'static, i16> {
```

Comment:
Please keep them sorted: `i16` should be between `i8` and `i32`

Reply:
My bad, I've corrected it